motionplan/cBiRRT.go

```diff
@@ -90,17 +90,17 @@ func newCBiRRTMotionPlanner(
 	}, nil
 }
 
-func (mp *cBiRRTMotionPlanner) plan(ctx context.Context,
-	goal spatialmath.Pose,
-	seed []referenceframe.Input,
-) ([]node, error) {
+func (mp *cBiRRTMotionPlanner) plan(ctx context.Context, goal spatialmath.Pose, seed []referenceframe.Input) ([]node, error) {
```

Comment:
when is [cBiRRT.go](https://github.com/viamrobotics/rdk/pull/3395/files#diff-0b42b846426b1b7936acc25176c074c0d8e830470f54ce737315deec3ab60fef) used vs tpSpace vs rrt?

Have we manually tested that the behavior of all of them has not regressed?

Reply:
I intend to test on arms before merging this